source/style.css

```diff
@@ -0,0 +1,100 @@
+/************* Reset css rules *************/
```

Comment:
I like your idea with signatures in the *.js classes. Can we use them here and in the html file too?